prow/cmd/deck/main.go

```diff
@@ -320,13 +320,26 @@ func main() {
 	runLocal := o.pregeneratedData != ""
 
 	var fallbackHandler func(http.ResponseWriter, *http.Request)
+	var pjListingClient pjListingClient
+	var githubClient deckGitHubClient
+	var gitClient git.ClientFactory
+	var podLogClients map[string]jobs.PodLogClient
 	if runLocal {
 		localDataHandler := staticHandlerFromDir(o.pregeneratedData)
 		fallbackHandler = localDataHandler.ServeHTTP
 
-		if o.spyglass {
-			initSpyglass(cfg, o, mux, nil, nil, nil)
+		var fjc fakePjListingClientWrapper
+		var pjs prowapi.ProwJobList
+		staticPjsPath := path.Join(o.pregeneratedData, "prowjobs.json")
+		content, err := ioutil.ReadFile(staticPjsPath)
+		if err != nil {
+			logrus.WithError(err).Fatal("Failed to read jobs from prowjobs.json.")
+		} else if err = json.Unmarshal(content, &pjs); err == nil {
```

Comment:
I think the 'else if' should just be a new block since an error from the previous command will exit the program anyways? (I wasn't expecting to see 'err == nil' here on first look).

Reply:
That's right, fixed